medical_patient_dob/models/medical_patient.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class MedicalPatient(models.Model):
+    _inherit = 'medical.patient'
+
+    @api.multi
+    def _format_dob(self):
+        self.ensure_one()
+        date = 'No DoB'
+        if self.dob:
+            date = fields.Datetime.from_string(self.dob).strftime('%m/%d/%Y')
```

Comment:
Hmmmm this isn't internationalized - Europe reverses the month and day aspects of this.

I feel like we should maybe use this instead: [convert_to_display_name](https://github.com/odoo/odoo/blob/9.0/openerp/fields.py#L1438). Pretty sure that will use the internal internationalization settings in order to display proper.